package.json

```diff
@@ -11,8 +11,8 @@
     "node": "9.3.0"
```

Comment:
We could probably update the version of Node.js to 12.16.0 ... 💭

Reply:
Perhaps it would be better to change the value to `>= 9.3.0`
https://docs.npmjs.com/files/package.json#engines
unless we have a reason to say it won't work on older versions than `12.16.0`?

And is this PR the right place for it? If so it could make sense to also bump `cypress` to the latest version (`4.0.2`). It doesn't really belong in this PR (in my mind, anyway), but I've already modified the cypress setup to make it work, and we've already bumped `http-server`. ¯\\\_(ツ)_/¯